spec/support/vcr_cassettes/lighthouse/auth/client_credentials/revoke_consent_204.yml

```diff
@@ -0,0 +1,31 @@
+---
+http_interactions:
+- request:
+    method: delete
+    uri: https://sandbox-api.va.gov/internal/auth/v3/user/consent?clientId=id&icn=123498767V234859
```

Comment:
This client id parameter needs to align with the rspec `DELETE` call for the necessary matching to occur.

`0oa2ey2m6kEL2897N2p7` is the client id in the rspec (rather than "id")